server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -75,12 +85,14 @@ export class RestGitService {
             `Created RestGitService: ${JSON.stringify({
                 "BaseUrl": storage.url,
                 "Storage-Routing-Id": this.getStorageRoutingHeaderValue(),
+                "Storage-Name": this.storageName,
```

Comment:
Why introduce another header ? Can we use the Storage-Routing-Id instead or we need both ?

Reply:
Got it. :)
Since our current C# code is treating storage-routing-id as tenant id and document id. If we need to add the storage name into the storage-routing-id, we need to change the method and definition for Storage-Routing-Id in C# to handle it.

Reply:
I think those 2 headers (storage-routing-id and storage-name) represent 2 different things. Storage-routing-id is the combination of tenantID+documentID which identifies where inside a storage account the data should be placed. storage-name represents the layer above - the storage account name to be used. In most of the cases, there will be a 1:1 mapping between tenantId and storage account name. But there might be situations later where the tenantId maps to multiple storage accounts. For now, I think it's appropriate to keep those 2 separate, and we can re-evaluate later if we can combine them somehow.